backend/src/main/java/com/step2020/server/managers/EventCreationManager.java

```diff
@@ -166,7 +166,9 @@ public void onCancelled(DatabaseError error) {
   }
 
   private boolean checkRequestInfoForRequiredKeys(Map<String, String> info) {
-    return !requestInfo.containsKey(UID) && !requestInfo.containsKey(EVENTID);  
+    // return !requestInfo.containsKey(UID) && !requestInfo.containsKey(EVENTID);  
```

Comment:
might as well clean up this comment since it just seems like a variable rename

Reply:
Addressing this in stand-up as we're unsure if this is a correct change or not now.... @Jkim-Hack